pkg/system/reconciler.go

```diff
@@ -77,6 +77,7 @@ type Reconciler struct {
 	AWSCloudCreds       *cloudcredsv1.CredentialsRequest
 	AzureCloudCreds     *cloudcredsv1.CredentialsRequest
 	AzureContainerCreds *corev1.Secret
+	NooBaaRootSecret    *corev1.Secret
```

Comment:
I think the name can be like the other secrets, also move this line to be with the other secrets...
```suggestion
	SecretRootMasterKey    *corev1.Secret
```